src/sql/workbench/api/common/extHostModelView.ts

```diff
@@ -613,10 +614,17 @@ class ComponentWrapper implements azdata.Component {
 		protected _id: string,
 		protected _logService: ILogService
 	) {
+		super();
 		this.properties = {};
 		this.itemConfigs = [];
 	}
 
+	public getDisposableEmitter<T>(): Emitter<T> {
```

Comment:
Maybe `getRegisteredEmitter`? The name here implies to me that it's just returning an emitter which is disposable (which it already is so seems kind of weird)